crates/update-bindings/src/main.rs

```diff
@@ -188,7 +188,7 @@ mod mapi_bindgen {
         winmd_path.push(WINMD_FILE);
         let mut source_path = get_out_dir();
         source_path.push("bindings.rs");
-        bindgen([
+        let _ = bindgen([
```

Comment:
clippy warning about ignored result, this function potentially returns a `Warnings` struct now, but it doesn't seem to affect the generated output